src/test/operators/join_test_runner.cpp

```diff
@@ -593,9 +674,9 @@ INSTANTIATE_TEST_SUITE_P(JoinHash, JoinTestRunner,
                          testing::ValuesIn(JoinTestRunner::create_configurations<JoinHash>()));
 INSTANTIATE_TEST_SUITE_P(JoinSortMerge, JoinTestRunner,
                          testing::ValuesIn(JoinTestRunner::create_configurations<JoinSortMerge>()));
+// TODO(anyone) #1852
```

Comment:
That's doesn't help a lot. Can you give a short description of what's the todo here?